hack/test-integration.sh

```diff
@@ -96,16 +101,39 @@ loop="${TIMES:-1}"
 # hack/test-integration.sh Template*
 # hack/test-integration.sh "(WatchBuilds|Template)"
 tests=( $(go run "${OS_ROOT}/hack/listtests.go" -prefix="${OS_GO_PACKAGE}/${package}.Test" "${testexec}" | grep -E "${1-Test}") )
+
+if [[ -n "${junit_report}" ]]; then
+	tmp_dir="/tmp/openshift/test-integration"
+	mkdir -p ${tmp_dir}
+	test_output_file="${tmp_dir}/test-go.log"
+	test_error_file="${tmp_dir}/test-go-err.log"
```

Comment:
We're not using this for anything in here -- if you are trying to replicate the checks we have in `hack/test-go.sh` for panics, etc -- factor those, too, into common functions.